src/game/AnimalManager.ts

```
@@ -466,155 +478,253 @@ export class AnimalManager {
+    );
+  }
+
+  private getNeighbors(animal: AnimalInstance): Neighbor[] {
+    const entries = this.spatialIndex.query(
+      animal.position.x,
+      animal.position.z,
+      animal.aiData.speciesConfig.neighborRadius,
+    );
+    const neighbors: Neighbor[] = [];
+    for (const entry of entries) {
+      if (entry.id === animal.id) continue;
```

Biome (2.4.4)

[error] Block statements are preferred in this position.

(lint/style/useBlockStatements)